### PR TITLE
Cleanups all around - and a feature

### PR DESCRIPTION
**Pull request by marcosps**
@fgyanz I've added a new argument to `scan` to search for a --conf, so it's easier for us to spot non supported configurations. Could you take a look to double check the details? I can merge it after you give you your input :)

### CODE REVIEW

**Comment by marcosps** [comment]
@fgyanz I solved the issue that you found, it was just checking the archs of the codestream. Just initialized it before checking if the codestream is missing, so this can be used to detect which packages to download.

File: klpbuild/config.py [hunk 1]
```diff
@@ -95,7 +92,6 @@ def setup_user_env(self, basedir):
 
     def load_user_conf(self):
         config = configparser.ConfigParser()
-        logging.info(f"Loading user configuration from '{self.user_conf_file}'")
```
Comment (fgyanz) [inline klpbuild/config.py:99]: This is kind of useful, no? I know that due to the current design we get the message several times per run, but still.

File: klpbuild/ksrc.py [hunk 2]
```diff
@@ -444,17 +444,12 @@ def get_patched_kernels(self, codestreams, commits, cve):
                 if not patched and kernel not in suse_tags:
                     continue
 
-                print(f"\n{cs.name()} ({kernel}):")
-
                 # If no patches/commits were found for this kernel, fallback to
                 # the commits in the main SLE branch. In either case, we can
                 # assume that this kernel is already patched.
-                for c in kern_commits if patched else suse_commits:
-                    print(f"{c}")
```
Comment (fgyanz) [inline klpbuild/ksrc.py:452]: Do we really need to remove this prints? It helps to track the logic taken by klp-build.

**Comment by marcosps** [comment]
I removed the last patch that reduced the output, so I believe that we improve that in a later stage. Merged!

**Pull request merged**